test/test_files/transaction/ddl/ddl.test

```diff
@@ -1,6 +1,56 @@
 -DATASET CSV tinysnb
 --
 
+-CASE DropNodeTablePropertyCommit
+-STATEMENT BEGIN TRANSACTION;
+---- ok
+-STATEMENT ALTER TABLE person DROP fName
+---- ok
+-STATEMENT COMMIT
+---- ok
+-STATEMENT MATCH (p:person) RETURN * ORDER BY p.ID LIMIT 1
+---- 1
+{_ID: 0:0, _LABEL: person, ID: 0, gender: 1, isStudent: True, isWorker: False, age: 35, eyeSight: 5.000000, birthdate: 1900-01-01, registerTime: 2011-08-20 11:25:30, lastJobDuration: 3 years 2 days 13:02:00, workedHours: [10,5], usedNames: [Aida], courseScoresPerTerm: [[10,8],[6,7,8]], grades: [96,54,86,92], height: 1.731000, u: a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11}
+
+-CASE DropNodeTablePropertyCommitRecovery
+-STATEMENT CALL auto_checkpoint=false;
+---- ok
+-STATEMENT BEGIN TRANSACTION;
+---- ok
+-STATEMENT ALTER TABLE person DROP fName
+---- ok
+-STATEMENT COMMIT
+---- ok
+-RELOADDB
+-STATEMENT MATCH (p:person) RETURN * ORDER BY p.ID LIMIT 1
+---- 1
+{_ID: 0:0, _LABEL: person, ID: 0, gender: 1, isStudent: True, isWorker: False, age: 35, eyeSight: 5.000000, birthdate: 1900-01-01, registerTime: 2011-08-20 11:25:30, lastJobDuration: 3 years 2 days 13:02:00, workedHours: [10,5], usedNames: [Aida], courseScoresPerTerm: [[10,8],[6,7,8]], grades: [96,54,86,92], height: 1.731000, u: a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11}
+
+-CASE DropNodeTablePropertyRollbackCommit
```

Comment:
Just noticed this. Can u drop the `Commit` suffix here?

Also other places in the test file with `RollbackCommit`.

Reply:
oops missed this while copying from the other branch